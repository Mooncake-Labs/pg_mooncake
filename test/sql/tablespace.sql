DROP TABLESPACE IF EXISTS mooncake_ts;
CREATE TABLESPACE mooncake_ts LOCATION '/tmp/tablespace_dir';
CREATE TABLE t (a int) USING columnstore TABLESPACE mooncake_ts;
SELECT table_name, regexp_replace(path, '[0-9]+/$', 'XXXX/') AS path FROM mooncake.columnstore_tables;
DROP TABLE t;
DROP TABLESPACE mooncake_ts;
