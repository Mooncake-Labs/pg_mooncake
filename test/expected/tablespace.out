DROP TABLESPACE IF EXISTS mooncake_ts;
NOTICE:  tablespace "mooncake_ts" does not exist, skipping
CREATE TABLESPACE mooncake_ts LOCATION '/home/postgres/tablespace_dir';
CREATE TABLE t (a int) USING columnstore TABLESPACE mooncake_ts;
SELECT table_name, regexp_replace(path, '[0-9]+/$', 'XXXX/') AS path FROM mooncake.columnstore_tables;
 table_name |                               path                                
------------+-------------------------------------------------------------------
 t          | /home/postgres/tablespace_dir/mooncake_contrib_regression_t_XXXX/
(1 row)

DROP TABLE t;
DROP TABLESPACE mooncake_ts;
